### PR TITLE
chore: default text is updated

### PR DESCRIPTION
**Pull request by palash-signoz**
Alert Description is updated

### CODE REVIEW

File: frontend/src/container/CreateAlertChannels/index.tsx [hunk 1]
```diff
@@ -21,26 +21,16 @@ function CreateAlertChannels({
...
+     *Alert:* {{ .Labels.alertname }}{{ if .Labels.severity }} - {{ .Labels.severity }}{{ end }}
 
+     *Summary:* {{ .Annotations.summary }}
      *Description:* {{ .Annotations.description }}
 
      *Details:*
        {{ range .Labels.SortedPairs }} • *{{ .Name }}:* {{ .Value }}
        {{ end }}
      {{ end }}`,
-		title: `[{{ .Status | toUpper }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}] {{ .CommonLabels.alertname }} for {{ .CommonLabels.job }}
```
Comment (prashant-shahi) [inline frontend/src/container/CreateAlertChannels/index.tsx:33]: @palash-signoz Is this removal intentional?

Comment (prashant-shahi) [reply frontend/src/container/CreateAlertChannels/index.tsx:33]: My bad for not clearly mentioning it. 😅️

The text I shared was for the placeholder for the description. Here, `text` key is used for that.
We use the `title` key for the placeholder for the title input box. We will need it back.

Comment (palash-signoz) [reply frontend/src/container/CreateAlertChannels/index.tsx:33]: NP _(with Axolo)_

**Review by prashant-shahi — APPROVED** [review]
:shipit: LGTM

**Review by ankitnayan — APPROVED** [review]
